juno/assetlist.json

```diff
@@ -552,6 +552,50 @@
         "logo_URIs": {
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/hopers.png"
         }
+      },
+      {
+        "description": "RED",
+        "type_asset": "cw20",
+        "address": "juno1g647t78y2ulqlm3lss8rs3d0spzd0teuwhdvnqn92tr79yltk9dq2h24za",
+        "denom_units": [
+          {
+            "denom": "cw20:juno1g647t78y2ulqlm3lss8rs3d0spzd0teuwhdvnqn92tr79yltk9dq2h24za",
+            "exponent": 0
+          },
+          {
+            "denom": "red",
+            "exponent": 6
+          }
+        ],
+        "base": "cw20:juno1g647t78y2ulqlm3lss8rs3d0spzd0teuwhdvnqn92tr79yltk9dq2h24za",
+        "name": "Red",
+        "display": "red",
+        "symbol": "RED",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/63efe91696e4153e92db1547b5d6ee618d673814/juno/images/red.jpeg"
```

Comment:
these should also be master branch, not /63ef...3814/. And then that should be all.